docs/mofed-container-env-vars.md

```diff
@@ -23,6 +23,7 @@ In most deployments its expected to not require setting those.
 | UNLOAD_STORAGE_MODULES |N|`"false"`| unload host storage modules prior to loading mofed modules  |
 | ENABLE_NFSRDMA |N|`"false"`| enable loading of nfs relates storage modules from mofed container|
 | RESTORE_DRIVER_ON_POD_TERMINATION |N|`"true"`| restore host drivers when container is gracefully stopped |
+| NVIDIA_NIC_DRIVERS_INVENTORY_PATH | N | `"/mnt/drivers-inventory"` | enable use of a persistent directory to store drivers' build artifacts to avoid recompilation between runs. Set to "" to disable |
```

Comment:
What will happen if user set this to `"/foo"` for example?

From the code it will always get the default value, no?

Reply:
> From the code it will always get the default value, no?

You are right and I'd say it's desired behavior. The only control option we'd like to expose is to disable the feature

Reply:
lets clarify documentation that changing the path has no effect when used with the operator